developer-portal/fast-api-integration-guide.md

```diff
@@ -0,0 +1,235 @@
+fast.addEventListener('user_event', (event) => {
+  console.log(`Got a user_event of type: ${event.name}`);
+  console.log(event);
+
+  // For example, when the user clicks the "Keep shopping" button after finalizing their order, you could redirect to an
+  // order confirmation page
+  if (event.name === 'Checkout - Order Completed') {
+    window.location = 'https://example.com/order-completed';
+  }
+});
+```
+
+🚨 In some environments, such as the embedded browsers inside of mobile apps, the Fast Checkout window will open in a separate browser and not be able to communicate these events back to your page. You should not rely on these events for mission-critical analytics and should always provide another way for buyers to see that their purchase is complete.
+
+# 4. Handle order creation in your backend
+
+Now we will set up your server to communicate with Fast Checkout.
+
+While users interact with the Fast Checkout application, Fast will send requests to your server for processing.
+
+[![Handle order creation in the backend](images/fast-api/handle-order-creation.png)](images/handle-order-creation.png)
+
+As users add items, apply coupons, and submit orders, Fast will send these events and changes in data to your server for further processing. See the full specification [here](https://www.notion.so/DRAFT-Public-BE-Reference-Checkout-Documentation-79f563ec12d04dbe807fe6f3f146e8ac).
+
+To start developing & testing your integration, you'll likely need to simulate some traffic to your machine. You can do this via our testing environment and developer tools.
+
+### Connecting to Fast Sandbox
+
+If you haven't already, begin onboarding here [https://sandbox.sdash.fast.co](https://sandbox.sdash.fast.co). 
+
+Complete the forms to complete your sandbox store onboarding. You will need to provide your the URL that your API is hosted at in the "**Install Fast Checkout**" section. Once you have registered your API and have a Fast credentials associated with your sandbox store, continue on to the next section.
+
+### Simulating test traffic
+
+To see what data our system will send you in production for a given payload, you'll need the fast developer tools.
+
+You can download the fast-cli developer tool here [https://github.com/fast-af/fast-test-suite-cli](https://github.com/fast-af/fast-test-suite-cli)
+
+The repository has detailed instructions on dependencies and usage, once you have everything installed you can run this command to run through a test scenario:
+
+```bash
+./fast-test-suite login
+./fast-test-suite pdp-simple-new-user --product-id=<product_id> --app-id=<app_id>
+```
+
+**NOTE:** You'll need to replace variables in the initial payload to match actual products that are registered in your system. We expect the seller to recognize the payload we pass. For full documentation on the possible options and full schema, see the reference documentation (coming soon).
```

Comment:
there are a few references to reference documentation (coming soon), is that something to be added here as well? Or does that already exist?

Reply:
I can go through and review for that. I believe that doc is hidden too, but I can clarify that. I am going to launch this but not add to sidebar until we get clarity on that other doc.